cmd/wit-bindgen-go/main_test.go

```diff
@@ -0,0 +1,42 @@
+// +build !tinygo
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"testing"
+)
+
+// TestSimpleGenVerbosity ensures that a basic generation case honors the verbose flag
+func TestSimpleGenVerbosity(t *testing.T) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd := makeCmd()
+	cmd.Writer = &stdout
+	cmd.ErrWriter = &stderr
+
+	args := []string{
+		"wit-bindgen-go",
+		"generate",
+		"--world",
+		"http-fetch-simple",
+		"--dry-run",
+		"../../testdata/codegen/simple-http.wit",
+	}
+
+	// Run the app without verbose
+	cmd.Run(context.Background(), args)
+	if stderr.Len() != 0 {
+		t.Errorf("output was written to stderr despite lack of --verbose")
+	}
+
+	stdout.Reset()
+	stderr.Reset()
+
+	// Run the app WITH verbose
+	cmd.Run(context.Background(), append(args, "--verbose"))
```

Comment:
**nit:**  I prefer a `-v` flag

Reply:
Ah so both are already possible -- `verbose` is a builtin (which comes with the `-v` alias automatically)